linkerd/dns/src/lib.rs

```diff
@@ -22,7 +23,8 @@ pub trait ConfigureResolver {
     fn configure_resolver(&self, _: &mut ResolverOpts);
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Error)]
+#[error("Invalid SRV record {:?}", self.0)]
```

Comment:
we've used lowercase elsewhere, mostly
```suggestion
#[error("invalid SRV record {:?}", self.0)]
```